metrics/cli/docs/features/diagrams/update_command_sequence_diagram.puml

```diff
@@ -0,0 +1,75 @@
+@startuml update_command_sequence_diagram
+
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+box Update Command
+participant MetricsCliRunner
+participant UpdateCommand
+participant Updater
+participant FirebaseService
+participant GitService
+participant NpmService
+participant FlutterService
+participant SentryService
+participant PathsFactory
+participant UpdateConfigFactory
+end box
+
+participant FileHelper
+
```

Comment:
:readable:
The diagram contains a lot of callings to different services that are not described in the document. Should we provide more details before introducing this diagram?